### PR TITLE
Finish translation of accessibilty

### PR DESCRIPTION
**Pull request by r3sMetz**
Add all recent translations to accessibilty.md

### CODE REVIEW

**Comment by r3sMetz** [comment]
long story short...

File: content/docs/accessibility.md [hunk 1]
```diff
@@ -444,77 +444,75 @@ The [eslint-plugin-jsx-a11y](https://github.com/evcohen/eslint-plugin-jsx-a11y)
...
+- [Den Barrierefreiheitsprüfer in OS X Safari nutzen](https://developer.apple.com/library/content/documentation/Accessibility/Conceptual/AccessibilityMacOSX/OSXAXTestingApps.html)
 
-### Screen readers {#screen-readers}
+### Screen-Reader {#screen-readers}
 
-Testing with a screen reader should form part of your accessibility tests.
+Das Testen mit einem Screen-Reader sollte bei deinen Barrierefreiheitstests nicht fehlen.
 
-Please note that browser / screen reader combinations matter. It is recommended that you test your application in the browser best suited to your screen reader of choice.
+Beachte dass es auf die Browser / Screen-Reader Kombination ankommt. Es empfiehlt sich dass du deine Applikation in dem Browser testest, welcher am besten zu deinem favorisierten Screen-Reader passt.
```
Comment (ph1p) [inline content/docs/accessibility.md]:
```suggestion
Beachte dass es auf die Browser / Screen-Reader Kombination ankommt. Es wird empfohlen, dass du deine Anwendung in dem Browser testest, der am besten zu deinem gewählten Screen-Reader passt.
```

File: content/docs/accessibility.md [hunk 2]
```diff
@@ -444,77 +444,75 @@ The [eslint-plugin-jsx-a11y](https://github.com/evcohen/eslint-plugin-jsx-a11y)
...
+- [Deque - JAWS Tastenkombinationen](https://dequeuniversity.com/screenreaders/jaws-keyboard-shortcuts)
 
-### Other Screen Readers {#other-screen-readers}
+### Weitere Screen-Reader {#other-screen-readers}
 
-#### ChromeVox in Google Chrome {#chromevox-in-google-chrome}
+#### ChromeVox im Google Chrome {#chromevox-in-google-chrome}
 
-[ChromeVox](https://www.chromevox.com/) is an integrated screen reader on Chromebooks and is available [as an extension](https://chrome.google.com/webstore/detail/chromevox/kgejglhpjiefppelpmljglcjbhoiplfn?hl=en) for Google Chrome.
+[ChromeVox](https://www.chromevox.com/) ist ein in Chromebooks integrierter ScreenReader und ist außerdem [als eine Erweiterung](https://chrome.google.com/webstore/detail/chromevox/kgejglhpjiefppelpmljglcjbhoiplfn?hl=de) für Chrome verfügbar.
```
Comment (ph1p) [inline content/docs/accessibility.md]:
```suggestion
[ChromeVox](https://www.chromevox.com/) ist ein in Chromebooks integrierter Screen-Reader und ist außerdem [als eine Erweiterung](https://chrome.google.com/webstore/detail/chromevox/kgejglhpjiefppelpmljglcjbhoiplfn?hl=de) für Chrome verfügbar.
```

File: content/docs/accessibility.md [hunk 3]
```diff
@@ -444,77 +444,75 @@ The [eslint-plugin-jsx-a11y](https://github.com/evcohen/eslint-plugin-jsx-a11y)
...
 
-Refer to the following guides on how best to use ChromeVox:
+Nutze die folgenden Hinweise, um zu erfahren wie ChromeVox am besten zu nutzen ist:
 
-- [Google Chromebook Help - Use the Built-in Screen Reader](https://support.google.com/chromebook/answer/7031755?hl=en)
-- [ChromeVox Classic Keyboard Shortcuts Reference](https://www.chromevox.com/keyboard_shortcuts.html)
+- [Google Chromebook Hilfe - Nutzung des Integrierten Screen-Readers](https://support.google.com/chromebook/answer/7031755?hl=de)
```
Comment (ph1p) [inline content/docs/accessibility.md]:
```suggestion
- [Google Chromebook Hilfe - Nutzung des integrierten Screen-Readers](https://support.google.com/chromebook/answer/7031755?hl=de)
```

File: content/docs/accessibility.md [hunk 4]
```diff
@@ -381,61 +381,61 @@ Jede Art von Widget hat ein spezielles Designpattern und sollte für User und de
...
-- [Everything About Color Contrast And Why You Should Rethink It](https://www.smashingmagazine.com/2014/10/color-contrast-tips-and-tools-for-accessibility/)
-- [A11yProject - What is Color Contrast](https://a11yproject.com/posts/what-is-color-contrast/)
+- [WCAG - Den Bedarf für Farbkontraste verstehen](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html)
+- [Alles über Farbkontraste und warum du darüber nachdenken](https://www.smashingmagazine.com/2014/10/color-contrast-tips-and-tools-for-accessibility/)
+- [A11yProject - Was sind Farbkontraste](https://a11yproject.com/posts/what-is-color-contrast/)
 
-It can be tedious to manually calculate the proper color combinations for all cases in your website so instead, you can [calculate an entire accessible color palette with Colorable](https://jxnblk.com/colorable/).
+Es kann sehr anstrengend sein die richtigen Farbkombinationen für alle Anwendungsfälle deiner Website manuell zu erstellen, anstatt dessen kannst du [eine vollständig barrierefreie Farbpalette mit Colorable berechnen](https://jxnblk.com/colorable/).
```
Comment (ph1p) [inline content/docs/accessibility.md]:
```suggestion
Es kann sehr anstrengend sein die richtigen Farbkombinationen für alle Anwendungsfälle deiner Webseite manuell zu erstellen, anstatt dessen kannst du [eine vollständig barrierefreie Farbpalette mit Colorable berechnen](https://jxnblk.com/colorable/).
```

File: content/docs/accessibility.md [hunk 5]
```diff
@@ -381,61 +381,61 @@ Jede Art von Widget hat ein spezielles Designpattern und sollte für User und de
...
-Indicate the human language of page texts as screen reader software uses this to select the correct voice settings:
+Gebe immer die, in den Texten deiner Seite, verwendete Sprache an, da Screen-Reader-Software diese Angabe nutzt um die richtigen Spracheinstellungen auszuwählen.
 
 - [WebAIM - Document Language](https://webaim.org/techniques/screenreader/#language)
 
-### Setting the document title {#setting-the-document-title}
+### Der Titel des Dokuments {#setting-the-document-title}
 
-Set the document `<title>` to correctly describe the current page content as this ensures that the user remains aware of the current page context:
+Verwende einen `<title>` welcher den aktuellen Inhalt der Seite korrekt beschreibt, um sicher zu gehen dass jeder User genau über den Kontext des Dokuments im Klaren ist.
```
[the remark this answers is not in the document]
Comment (r3sMetz) [reply content/docs/accessibility.md]: Please write "korrekt zu beschreiben, ... "

[the remark this answers is not in the document]
Comment (ph1p) [reply content/docs/accessibility.md]: done ;)

File: content/docs/accessibility.md [hunk 6]
```diff
@@ -381,61 +381,61 @@ Jede Art von Widget hat ein spezielles Designpattern und sollte für User und de
...
+## Entwicklungs- und Testingtools {#development-and-testing-tools}
 
-There are a number of tools we can use to assist in the creation of accessible web applications.
+Es gibt eine Reihe von Werkzeugen um dich bei der Erstellung barrierefreier Webapplikationen zu unterstützen. 
 
-### The keyboard {#the-keyboard}
+### Die Tastatur {#the-keyboard}
 
-By far the easiest and also one of the most important checks is to test if your entire website can be reached and used with the keyboard alone. Do this by:
+Das bei Weitem Einfachste, aber auch eines der wichtigsten Dinge die du prüfen musst ist die Bedienbarkeit deiner vollständigen Website außschließlich via Tastatur. Teste dies wie folgt:
```
[the remark this answers is not in the document]
Comment (r3sMetz) [reply content/docs/accessibility.md]: Please remove one "und" at "einfachste und und"^^

File: content/docs/accessibility.md [hunk 7]
```diff
@@ -444,77 +444,75 @@ The [eslint-plugin-jsx-a11y](https://github.com/evcohen/eslint-plugin-jsx-a11y)
   }
   ```
 
-### Testing accessibility in the browser {#testing-accessibility-in-the-browser}
+### Barrierefreiheit im Browser testen {#testing-accessibility-in-the-browser}
 
-A number of tools exist that can run accessibility audits on web pages in your browser. Please use them in combination with other accessibility checks mentioned here as they can only
-test the technical accessibility of your HTML.
+Es gibt eine Reihe von Werkzeugen um die Barriefreiheit deiner Webseiten im Browser zu kontrollieren. Bitte nutze Diese nur in Verbindung mit den bereits genannten Werkzeugen, da diese Tests nur die technische Barrierefreiheit in deinem HTML kontrollieren können.
```
[the remark this answers is not in the document]
Comment (r3sMetz) [reply content/docs/accessibility.md]: Please remove "auf" after "Barrierefreiheitsprüfungen" or write "auf deinen Webseiten"

File: content/docs/accessibility.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (ph1p) [reply content/docs/accessibility.md]: done 👍

File: content/docs/accessibility.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (ph1p) [reply content/docs/accessibility.md]: done (:

**Comment by r3sMetz** [comment]
Think we're done here :)

**Comment by ph1p** [comment]
> Think we're done here :)

Yep! Well done! Thank you :)

**Pull request merged**